packages/core/__tests__/login.test.ts

```diff
@@ -0,0 +1,17 @@
+import { RedoclyClient } from '../src/redocly';
+
+describe('loginnnnnn', () => {
```

Comment:
too many nnnnnn )

Reply:
I think this test should be moved into packages/core/src/redocly/__tests__ folder